Libraries/StyleSheet/StyleSheetTypes.js

```diff
@@ -232,6 +266,23 @@ type ____LayoutStyle_Internal = $ReadOnly<{
    */
   padding?: DimensionValue,
 
+  /** Setting `paddingBlock` is like setting both of
+   *  `paddingTop` and `paddingBottom`.
+   */
+  paddingBlock?: DimensionValue,
+
+  /** `paddingBlockEnd` works like `padding-bottom` in CSS.
+   * See https://developer.mozilla.org/en-US/docs/Web/CSS/padding-bottom
+   * for more details.
```

Comment:
nit: we can link to the 1:1 prop. E.g. https://developer.mozilla.org/en-US/docs/Web/CSS/padding-block-end